brjs-core/src/main/java/org/bladerunnerjs/model/NodeImporter.java

```diff
@@ -205,22 +205,30 @@ private static void findAndReplaceInTextFiles(BRJS brjs, Collection<File> files,
 	{
 		for (File f : files) {
 			if (f.length() != 0) {
-				if (checkFileMimeType(f).startsWith("text")) {
-					findAndReplaceInTextFile(brjs, f, sourceRequirePrefix, targetRequirePrefix);
+				try {
+					if (checkFileMimeType(f).startsWith("text")) {
+						findAndReplaceInTextFile(brjs, f, sourceRequirePrefix, targetRequirePrefix);
+					}
+				}
+				catch (MagicMatchNotFoundException e) {
+					continue;
 				}
 			}
 		}
 	}
 
-	private static String checkFileMimeType(File file) throws IOException {
+	private static String checkFileMimeType(File file) throws IOException, MagicMatchNotFoundException {
 		byte[] data = Files.readAllBytes(Paths.get(file.getAbsolutePath()));
```

Comment:
This may become very inefficient. It might be worth not checking based on mime type and having a list if known extensions we filter. While it's a list we have to maintain we won't need to read every file in the app to figure out whether we need to read it again and replace the content.